reward/rewardManager.go

```diff
@@ -34,3 +41,111 @@ type governanceHelper interface {
 func isEmptyAddress(addr common.Address) bool {
 	return addr == common.Address{}
 }
+
+type RewardManager struct {
+	sm  *stakingManager
+	rcc *rewardConfigCache
+	gh  governanceHelper
+}
+
+func NewRewardManager(bc *blockchain.BlockChain, gh governanceHelper) *RewardManager {
+	return &RewardManager{
+		sm:  newStakingManager(bc, gh),
+		rcc: newRewardConfigCache(gh),
+		gh:  gh,
+	}
+}
+
+// getTotalTxFee returns the total transaction gas fee of the block.
+func (rm *RewardManager) getTotalTxFee(header *types.Header, rewardConfig *rewardConfig) *big.Int {
+	totalGasUsed := big.NewInt(0).SetUint64(header.GasUsed)
+	totalTxFee := big.NewInt(0).Mul(totalGasUsed, rewardConfig.unitPrice)
```

Comment:
Can we use just one `big.Int` instead of declaring two different `big.Int` variables?

Reply:
reflected